packages/ses/docs/reference.md

```diff
@@ -0,0 +1,309 @@
+# Endo and Hardened JavaScript (SES) Programming Reference
+
+This document describes how Hardened JavaScript (formerly SES) affects writing
+Agoric JavaScript code.
+It is very much a "how to do something" document, with little explanation about why and
```

Comment:
the howto and reference genres are essentially disjoint, no?

_I suppose such comments are orthogonal to this PR, which is just a bulk move operation._